msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -30,7 +30,7 @@ RSC permissions are categorized based on:
 | **Resource type** |  &nbsp; | &nbsp; |
 | Team | ✔️ | ✔️ |
 | Chat or meeting | ✔️ | ✔️ |
-| User | NA | ✔️ |
+| User | ✔️ | NA |
```

Comment:
@v-bvishnu  Sorry about back and forth on this one, but shouldn't the delegated column also be checked? I can see some .User RSC permissions in the table at the bottom of this page which are delegated only.

Reply:
Updated.